CHANGELOG/CHANGELOG-1.14.md

```diff
@@ -0,0 +1,88 @@
+# Changelog 1.14
+
+## 1.14.0 - 2023-10-27
+
+### Added
+
+- Add rate-limit config to Egress to specify the rate limit of north-south egress traffic of this Egress. ([#5425](https://github.com/antrea-io/antrea/pull/5425), [@GraysonWu])
+- Add `IPAllocated` and `IPAssigned` conditions to Egress status to improve Egress visibility. ([#5282](https://github.com/antrea-io/antrea/pull/5282), [@AJPL88] [@tnqn])
+- Add L7 visibility support to enables visibility into the layer 7 protocols in Antrea. ([#5218](https://github.com/antrea-io/antrea/pull/5218), [@tushartathgur])
+- Add goroutine stack dump in support bundle for both agent and controller. ([#5538](https://github.com/antrea-io/antrea/pull/5538), [@aniketraj1947])
+- Add "X-Load-Balancing-Endpoint-Weight" header to AntreaProxy Service healthcheck. ([#5299](https://github.com/antrea-io/antrea/pull/5299), [@hongliangl])
+- Add log rotation configuration in antrea-agent config for audit logs. ([#5337](https://github.com/antrea-io/antrea/pull/5337) [#5366](https://github.com/antrea-io/antrea/pull/5366), [@antoninbas] [@mengdie-song])
+- Add GroupMembers API Pagination support for Client. ([#5533](https://github.com/antrea-io/antrea/pull/5533), [@qiyueyao])
+- Add Namespaced Group Membership API for Antrea Controller. ([#5380](https://github.com/antrea-io/antrea/pull/5380), [@qiyueyao])
+- Support Pod secondary interfaces on VLAN network, and add OVS bridge creation for the secondary network. ([#5341](https://github.com/antrea-io/antrea/pull/5341) [#5365](https://github.com/antrea-io/antrea/pull/5365) [#5279](https://github.com/antrea-io/antrea/pull/5279), [@jianjuns])
+- Add FlexibleIPAM support for Antrea Multicast. ([#4922](https://github.com/antrea-io/antrea/pull/4922), [@ceclinux])
+- Enable Windows OVS container to run on pristine host environment. ([#5440](https://github.com/antrea-io/antrea/pull/5440), [@NamanAg30])
+- Update `Install-WindowsCNI-Containerd.ps1` script to make it compatible with containerd 1.7. ([#5528](https://github.com/antrea-io/antrea/pull/5528), [@NamanAg30])
+- Add checksum of Windows config files as annotations to Windows manifests to avoid stale ConfigMaps. ([#5545](https://github.com/antrea-io/antrea/pull/5545), [@Atish-iaf])
```

Comment:
> Remove auto-generated suffix from ConfigMap names, and add config checksums as Deployment annotations in Windows manifests, to avoid stale ConfigMaps when updating Antrea while preserving automatic rolling of Pods.

Reply:
Done, thanks.